tests/e2e/config/bootstrap.js

```diff
@@ -78,25 +79,49 @@ function observeConsoleLogging() {
 			return;
 		}
 
```

Comment:
The rpevious version of this function was copied from an earlier version of Gutenberg, and the changes here are just a copy+paste of the latest version from Gutenberg.